doc/modules/impute.rst

```diff
@@ -30,7 +30,7 @@ that contain the missing values::
     >>> from sklearn.impute import SimpleImputer
     >>> imp = SimpleImputer(missing_values=np.nan, strategy='mean')
     >>> imp.fit([[1, 2], [np.nan, 3], [7, 6]])       # doctest: +NORMALIZE_WHITESPACE
-    SimpleImputer(copy=True, fill_value=None, missing_values=nan, strategy='mean', verbose=0)
+    SimpleImputer(add_indicator=False, copy=True, fill_value=None, missing_values=nan, strategy='mean', verbose=0)
```

Comment:
seeing as you already have `NORMALIZE_WHITESPACE`, please wrap this to two lines to avoid exceeding 80 characters per line.